test.js

```diff
@@ -73,8 +74,31 @@ function createMultiLineSourceMapWithSourcesContent() {
   return sourceMap;
 }
 
-function compareStackTrace(sourceMap, source, expected) {
-  // Check once with a separate source map
+function createMultiLineSourceMapWithFileUrl() {
+  var sourceMap = new SourceMapGenerator({
+    file: '.generated.js',
+    sourceRoot: fileURL('.')
+  });
+  for (var i = 1; i <= 100; i++) {
+    sourceMap.addMapping({
+      generated: { line: i, column: 0 },
+      original: { line: 1000 + i, column: 99 + i },
+      source: 'line' + i + '.js'
+    });
+  }
+  return sourceMap;
+}
+
+
+function fileURL(file) {
+    file = path.resolve(file);
+    file = file.replace(/\\/g, '/');
+    file = file[0] !== '/' ? '/' + file : file;
+    return encodeURI('file://' + file);
```

Comment:
Can you update this (and the other new functions) to follow the 2 space indentation that the rest of the project uses?